pkg/util/containerd/namespaces.go

```diff
@@ -17,9 +19,33 @@ import (
 // "containerd_namespace" option has been set, it returns that namespace.
 // Otherwise, it returns all of them.
 func NamespacesToWatch(ctx context.Context, containerdClient ContainerdItf) ([]string, error) {
-	if namespace := config.Datadog.GetString("containerd_namespace"); namespace != "" {
-		return []string{namespace}, nil
+	if namespaces := config.Datadog.GetString("containerd_namespace"); namespaces != "" {
```

Comment:
`GetStringSlice` should do that for you.

Reply:
Changed this to `GetStringSlice` to be consistent with the other options :+1:
That means that now namespaces need to be separated by spaces and not by commas.